xml/security_ldap.xml

```diff
@@ -398,266 +399,195 @@ objectclass (2.16.840.1.113730.3.2.333 NAME 'nsPerson' <co xml:id="co-ldap-schem
   </variablelist>
 
   <sect2 xml:id="sec-security-ldap-server-instance" >
-  <title>Creating the &ds389; Instance</title>
+  <title>Creating a &ds389; Instance with an .inf File</title>
     <para>
-     You create the instance with the <command>dscreate</command> command.
-     It can take a configuration file (<filename>*.inf</filename>) which
-     defines the instance configuration settings. Alternatively, the command can
-     be run interactively.
+     Use the <command>dscreate</command> command to create a new &ds389; instance.
+     There are several ways to create a new instance: interactively, from a 
+     template file, using &yast;, and using a custom configuration file.
+     In this section you will learn how to use a custom configuration file. This 
+     file must have an <filename>.inf</filename> extension. 
```

Comment:
WEEEEELLLLLL it doesn't "have to" be a .inf. It just has to be inf formatted, but you could call it .config for all the command cares. so maybe "should" rather than "must"?

Reply:
cool I will clarify.